### PR TITLE
Added contributors list.

### PR DESCRIPTION
**Pull request by JPYamamoto**
Created the Contributors list, and added the names of the people who have already contributed to this project.
Also linked to it from the Contributing and Readme files.
Solving #74

### CODE REVIEW

**Review by IanGarciaUnam — APPROVED** [review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Contributors
+
+This project is the result of the dedicated efforts of many people. It is because of their generous contributions that open-source project like this one are alive.
```
Comment (paulgoetze) [inline CONTRIBUTORS.md:3]: I think that would be the plural "open-source project**s**", right?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This is the place to recognize their work.
+___
+
+- **Paul Götze**: Creator of *adoptoposs*.
+	- [GitHub](https://github.com/paulgoetze)
+	- [Twitter](https://twitter.com/paulgoetze)
+- **Enitan Williams**: Contributor.
+	- [GitHub](https://github.com/akhanaton)
+- **Florian Kraft**: Contributor.
```
Comment (paulgoetze) [inline CONTRIBUTORS.md:13]: Would you like to be mentioned as co-maintainer, @floriank? Might also make it more clear who can be addressed regarding any concerns etc.

**Review by paulgoetze — CHANGES_REQUESTED** [review]
Thanks a lot @JPYamamoto, I love the wording! Only found one small typo.
Let's wait for what @floriank want's to be mentioned as, then this should be good to be merged.

**Comment by floriank** [comment]
So far my only contributions are reviews, but if no one has a problem with me being mentioned that way, ship it ❤️

Merci beaucoup. @JPYamamoto

**Comment by JPYamamoto** [comment]
Made the requested modifications. Let me know if there's anything else that needs changing :)

File: CONTRIBUTORS.md (hunk 1 above)
Comment (JPYamamoto) [reply CONTRIBUTORS.md]: Fixed

**Review by paulgoetze — APPROVED** [review]
Looks good, thanks!

**Pull request merged**